client/html/templates/catalog/filter/search-body-default.php

```diff
@@ -9,7 +9,7 @@
 $enc = $this->encoder();
 $phrase = $enc->attr( $this->param( 'f_search' ) );
 $name = $enc->attr( $this->formparam( 'f_search' ) );
-$hint = $this->translate( 'client', 'Please enter at least three characters' );
+$hint = $this->translate( 'client', 'Please enter at least two characters' );
```

Comment:
Please leave that as is. You can overwrite translations easily in your installation to match your setting